test/integration/vaultdynamicsecret/terraform/postgres.tf

```diff
@@ -8,6 +8,11 @@ resource "helm_release" "postgres" {
   wait             = true
   wait_for_jobs    = true
 
+  set {
```

Comment:
Should this be conditional on scale testing?

Reply:
done here: https://github.com/hashicorp/vault-secrets-operator/pull/916/commits/a8668608c870f969a51087ffe88a018c88d15bf9